src/Microsoft.AspNetCore.Cors/Infrastructure/CorsService.cs

```diff
@@ -133,6 +168,7 @@ public virtual void EvaluatePreflightRequest(HttpContext context, CorsPolicy pol
                 !requestHeaders.All(header => CorsConstants.SimpleRequestHeaders.Contains(header, StringComparer.OrdinalIgnoreCase) ||
```

Comment:
Does it make sense to refactor this a little, so we can log exactly which request header is not supported by the CORS policy? Maybe a foreach? @rynowak @kichalla